draft-ietf-dnsop-domain-verification-techniques.md

```diff
@@ -213,11 +213,9 @@ The ACME example in {{txt-based}} is implemented by Let's Encrypt {{LETSENCRYPT}
 
 {{GOOGLE-WORKSPACE-TXT}} asks the user to sign in with their administrative account and obtain their verification token as part of the setup process for Google Workspace. The verification token is a 68-character string that begins with "google-site-verification=", followed by 43 characters. Google recommends a TTL of 3600 seconds. The owner name of the TXT record is the domain or subdomain neme being verified.
 
-{{GOOGLE-WORKSPACE-CNAME}} lets you specify a CNAME record for verifying domain ownership. The user gets a unique 12-character string that is added as "Host", with TTL 3600 (or default) and Destination an 86-character string beginning with "gv-" and ending with ".domainverify.googlehosted.com.".
```

Comment:
This is in the "TXT" section; the same content already appears in the "CNAME" section later on, and I don't see a reason to duplicate it here.